pyprof/nvtx/nvmarker.py

```diff
@@ -68,14 +68,57 @@ def isfunc(mod, f):
     return ins.ismethod(attr) or ins.isfunction(attr) or ins.ismethoddescriptor(attr) or ins.isbuiltin(attr)
 
 
-def traceMarker(stack):
+def traceMarker():
+    # Returns a string representing the stack of function calls separated with '/'
+    #
+    def get_func_stack():
+        func_stack = ""
```

Comment:
Do you need to predefine this here?
All you are doing on line 101 is concatenating '/' and the built up frame info.

Reply:
Yes I do. Line 101 always adds func_stack to itself, so it has to be initially defined or else it will error.

Reply:
Yes but func_stack is "" to start and never has anything added to it before line 101. 
Line 101 can just become `func_stack = "/" + frame_info

Reply:
No, it can't. It is inside of a loop.

Reply:
Nevermind. I misread the context.